docs/src/content/docs/guides/connectionoptions.mdx

```diff
@@ -0,0 +1,183 @@
+
+1. **Receive Inbound Twilio Phone call** → Use the `joinUrl` with a Twilio [`<Stream>`](https://www.twilio.com/docs/voice/twiml/stream).
+
+    ```xml
+    <!-- Example TwiML Response -->
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Response>
+        <Connect>
+            <Stream url="your_ultravox_join_url" />
+        </Connect>
+    </Response>
+
+    ```
+</Steps>
+
+The above shows how to create a TwiML Bin and use that for handling the inbound call. Consult the [Twilio docs](https://www.twilio.com/docs) for more on all the options Twilio provides for handling phone calls.
+
+## WebSocket Integration
+
+:::caution[Server-to-Server Only]
+WebSocket connections are designed for server-to-server communication. For browser or mobile applications, use our client SDKs with WebRTC for optimal performance. WebSocket connections over TCP can experience audio blocking and ordering constraints that make them unsuitable for direct client use.
+:::
+
+### Creating a WebSocket Call
+
+Creating a WebSocket-based call with Ultravox requires setting `medium` to `serverWebSocket` and passing in parameters for sample rates and buffer size.
+
+- **inputSampleRate** (required): Sample rate for input (user) audio (e.g., 48000).
+- **outputSampleRate** (optional): Sample rate for output (agent) audio (defaults to inputSampleRate).
+- **clientBufferSizeMs** (optional): Size of the client-side audio buffer in milliseconds. Smaller buffers allow for faster interruptions but may cause audio underflow if network latency fluctuates too greatly. For the best of both worlds, set this to some large value (e.g. 30000) and implement support for [PlaybackClearBuffer](../datamessages#websocket-specific) messages. (Defaults to 60).
+
+### Example: Creating an Ultravox Call with WebSockets
+```javascript
+    const response = await fetch('https://api.ultravox.ai/api/calls', {
+        method: 'POST',
+        headers: {
+            'X-API-Key': 'your_api_key',
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            systemPrompt: "You are a helpful assistant...",
+            model: "fixie-ai/ultravox",
+            voice: "Mark",
+            medium: {
+                serverWebSocket: {
+                    inputSampleRate: 48000,
+                    outputSampleRate: 48000,
+                    clientBufferSizeMs: 30000
+                }
+            }
+        })
+    });
+
+    const { joinUrl } = await response.json();
+```
+
+### Example: Joining a Call with Websockets
+```python
+import websockets
+
+socket = await websockets.connect(join_url)
+audio_send_task = asyncio.create_task(_send_audio(socket))
+async for message in socket:
+    if isinstance(message, bytes):
+        # Handle agent audio data
+    else:
+        # Handle data message. See *insert link to /datamessages*.
```

Comment:
Can you add this link please?

Reply:
can't add a link in the code but added above